.github/workflows/nmodl-ci.yml

```diff
@@ -0,0 +1,118 @@
+        shell: bash
+
+      - name: Set up Python3
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+        env:
+          PYTHON_VERSION:  ${{ env.DEFAULT_PY_VERSION }}
+       
+      - name: Install Python3 dependencies
+        run: |
+          pip3 install -U pip setuptools scikit-build Jinja2 PyYAML pytest 'sympy>=1.3,<1.6'
+          
+      - uses: actions/checkout@v2
+        
+      - name: Build and Test
+        id: build-test
+        shell: bash
+        working-directory: ${{runner.workspace}}/nmodl
+        run:  |
+          if [[ "${{ startsWith(matrix.os, 'macOS') }}" = "true" ]]; then
+              export PATH=/usr/local/opt/flex/bin:/usr/local/opt/bison/bin:$PATH;
+              export CXX=g++;
+              export CC=gcc;
+          fi
+          if [[ "${{ startsWith(matrix.os, 'ubuntu') }}" = "true" ]]; then
+              export CXX=c++;
+              export CC=cc;
+          fi
+          echo "------- Build, Test and Install -------"
+          mkdir build && pushd build
+          cmake .. -DPYTHON_EXECUTABLE=$(which python3) -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR
+          make;
+          if [ $? -ne 0 ]; then
+            make VERBOSE=1;
+          fi
+          ctest -VV
+          make install
+         
+        env:
+          INSTALL_DIR: ${{ runner.workspace }}/install
+
+      # This step will set up an SSH connection on tmate.io for live debugging.
+      # To trigger it, simply prefix your branch name with `live-debug`
+      #   i.e.: live-debug-hocfix
+      - name: live debug session on failure
+        if: failure() && startsWith(github.ref, 'refs/heads/live-debug')
+        uses: mxschmitt/action-tmate@v3
+
+      - name: Documentation 
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        id: documentation
+        working-directory: ${{runner.workspace}}/nmodl
+        run: |
+          echo "------- Build Documentation -------";
+          python3 setup.py build_ext --inplace docs -j 2 -G "Unix Makefiles";
+          cd _skbuild/linux-x86_64-3.8/setuptools/sphinx;
+          rm -rf doctest doctrees && touch .nojekyll;
+          echo "<meta http-equiv=\"refresh\" content=\"0; url=./html/index.html\" />" > index.html;
+          echo ::set-output name=status::done
+          
+#       - name: Deploy 🚀
+#         uses: JamesIves/github-pages-deploy-action@3.7.1
+#         if: steps.documentation.outputs.status == 'done' && startsWith(github.ref, 'refs/heads/master')
+#         with:
+#           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#           BRANCH: gh-pages # The branch the action should deploy to.
+#           FOLDER: ${{runner.workspace}}/nmodl/_skbuild/linux-x86_64-3.8/setuptools/sphinx # The folder the action should deploy.
+#           CLEAN: false # Automatically remove deleted files from the deploy branch
```

Comment:
@alexsavulescu : should we uncomment this? where this secrets needs to be added? Just need to create in Github settings? You have now admin permissions, so go ahead with necessary settings.

Reply:
If we drop Travis, then we can uncomment this. Otherwise no point in pushing twice to gh-pages

Reply:
If GitHub actions work, we can remove documentation part from Travis.
(Anyway, travis will be removed soon)

Reply:
Done